operations/iuf/examples/iuf_activity.md

```diff
@@ -0,0 +1,57 @@
+# `iuf activity` Examples
```

Comment:
should probably get new output in here with the functioning Status and your column changes

Reply:
yeah, that will be an ongoing issue for a while as both the -h output and the stdout is changing quite a bit.  I am thinking when all other comments are done, I'll decide if I update them in this PR or if I commit and then do a second PR to update all of those.

Reply:
fixed